aman/src/pages/ForgotPassword.jsx

```diff
@@ -0,0 +1,70 @@
+import { z } from "zod";
+import { useState } from "react";
+import Input from "../components/Input";
+import Button from "../components/Button";
+import Error from "../components/Error";
+import Label from "../components/Label";
+
+const ForgotPassword = ({ handleClick }) => {
+  const [formData, setFormData] = useState({ email: "" });
+  const [error, setError] = useState({});
+  const schema = z.object({
+    email: z.string().email("Must be a valid email address"),
+  });
+
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const result = schema.safeParse({
+      ...formData,
+    });
+    if (!result.success) {
+      const formattedErrors = result.error.format();
+      setError(formattedErrors);
+    } else {
+      setError({});
+    }
+  };
+  return (
+    <div className="mb-32">
+      <div className="flex flex-col p-6">
```

Comment:
In this div both the p tags have text centre so instead of mentioning it twice mention text centre in the parent div

Reply:
Done